09-hypothesis-testing.Rmd

```diff
@@ -761,18 +761,20 @@ Using our shorthand interpretation for 95% confidence intervals from Subsection
 visualize(bootstrap_distribution) + 
   shade_confidence_interval(endpoints = percentile_ci)
 ```
-```{r bootstrap-distribution-two-prop-percentile, echo=FALSE, fig.show='hold', fig.cap="Percentile-based 95\\% confidence interval.", purl=FALSE, fig.height=3.2}
+```{r bootstrap-distribution-two-prop-percentile, echo=FALSE, fig.show="hold", fig.cap="Percentile-based 95\\% confidence interval.", purl=FALSE, fig.height=3.2}
 # Will need to make a tweak to the {infer} package so that it doesn't always display "Null" here (added to `develop` branch on 2019-10-26)
```

Comment:
@rudeboybert We can probably remove this extra message and the commented out `ggtitle()` below, right?

Reply:
Yep. After #354, I plan on doing on last final sweep to remove such remaining comments.